anghammarad/src/main/scala/com/gu/anghammarad/messages/HangoutsService.scala

```diff
@@ -2,30 +2,31 @@ package com.gu.anghammarad.messages
 
 import com.gu.anghammarad.AnghammaradException.Fail
 import com.gu.anghammarad.models.HangoutMessage
-import lol.http.{Client, Post}
 
-import scala.concurrent.duration._
 import scala.util.{Success, Try}
-
+import sttp.client3._
 
 object HangoutsService {
-  // using a blocking call, but lolhttp still requires an execution context
-  import scala.concurrent.ExecutionContext.Implicits.global
 
-  val client = Client("chat.googleapis.com", 443, "https")
+  def postMessage(webhook: String, message: String): Response[Either[String, String]] = {
+    val backend = HttpURLConnectionBackend()
+    basicRequest
+      .body(message)
+      .post(uri"$webhook")
```

Comment:
What's `uri` here?

Reply:
This is shorthand. It give the same result as URI.parse(webhook)
From the docs:
Parse the given string (with embedded expressions) as an uri.
Any values embedded in the URI using the ${...} syntax will be URL-encoded, taking into account the context. Parts of the URI given as literal strings (not embedded values), are assumed to be URL-encoded and thus will be decoded when creating the Uri instance.

I can add a comment to the code if that helps?